CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+## Unreleased
+### Changed
+- Custom `sdist` command extends from `setuptools` instead of `distutils`. [#125](https://github.com/PyO3/setuptools-rust/pull/125)
```

Comment:
I suggest describing what the effect is to the user, rather than implementation details. Maybe something like the below?

```suggestion
### Fixed
- Fix some files unexpectedly missing from `sdist` command output. [#125](https://github.com/PyO3/setuptools-rust/pull/125)
```

Reply:
Suggestion applied, thanks!